serpent/oecd-fullcore

```diff
@@ -27,11 +26,15 @@ sic    0.03825
 pyc    0.04225
 matrix
 
+particle 2
+
+bkrnl  0.01000
+buffr2 0.01180
+pyc2   0.01410
+matrix2
+
 /*************************
 *     Fuel column        *
-* Radius 0.6223 cm       *
-* Length 79.3 cm         *
-* Packing Fraction 0.35  *
```

Comment:
Did you want to replace this with anything? Metrics for the oecd benchmark design?

Reply:
These are comments, so future users know the dimensions of some components in the reactor.
I guess I deleted it because every time I was updating the file I had to update the comments as well and I got tired of doing it. I could add them back if you think it is better that way.

Reply:
I do like them in there, as someone not writing the code. But I also understand why you left them out. If you won't need to be changing the values anytime soon, then please add it back in. If you will be changing them then you can do that with another PR later once it is not changing.

Reply:
Cool. Done.